deep_qa/data/instances/sequence_tagging/verb_semantics_instance.py

```diff
@@ -0,0 +1,173 @@
+from typing import Dict, List, Tuple
+
+import numpy
+from overrides import overrides
+
+from ..instance import TextInstance, IndexedInstance
+from ...data_indexer import DataIndexer
+
+
+class VerbSemanticsInstance(TextInstance):
+    """
+    A VerbSemanticsInstance is a :class:`TextInstance` that is a single sentence of text,
+    where that sentence has some associated categorical label.
+    """
+    def __init__(self, sentence: List[str], verb: (int, int), entity: (int, int), state_change: str, arg1: (int, int), arg2: (int, int), index: int=None):
+        """
+        sentence: a sentence from which we want to predict verb semantics.
+        A sentence can mention multiple verbs and entities, a data instance
+        is talking about a particular (verb, entity) pair in a given sentence
+        verb: span of the verb indicating the event being referred to in the query
+        entity: entity indicating 
+        """
+        """ TODO: initialize all members of the class"""
+        self.sentence = sentence
+        self.verb = verb
+        self.entity = entity
+        tags = self.__make_tag_sequence(arg1, arg2)
+        super(VerbSemanticsInstance, self).__init__((state_change, tags), index)
+
+
+    def __str__(self):
+        return 'VerbSemanticsInstance(' + self.sentence + ', ' + str(self.label) + ')'
+
+    def __make_tag_sequence(self, arg1: (int, int), arg2: (int, int)) -> List[str]:
+        """
+        Convert (arg1, arg2) into a tag sequence over the sentence
+        :param arg1: startIndex,endIndex of arg1 in the sentence 
```

Comment:
Again here, because we use sphinx + numpydoc to generate our documentation, this should be formatted slightly differently, like this: https://github.com/allenai/deep_qa/blob/6a0039822f9f2c727151b6143e2214d331bdc111/deep_qa/tensors/backend.py#L153-L167.